ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -420,8 +458,8 @@ boolean checkConnectionPlain(String url, HiveConf hiveConf) {
   }
 
   @Override
-  public List<RangerPolicy> addDenyPolicies(List<RangerPolicy> rangerPolicies, String rangerServiceName,
-                                            String sourceDb, String targetDb) throws SemanticException {
+  public RangerPolicy getDenyPolicyForReplicatedDb(String rangerServiceName,
```

Comment:
are u using the source db param

Reply:
yes, while defining ranger policy name.

Reply:
Yes, it is being used while specifying the policy  name.